README.md

```diff
@@ -64,7 +64,7 @@ Download the binary from [releases](https://github.com/src-d/code-annotation/rel
 
 The pieces of code to be labeled are called _file pairs_. They must be provided via an [SQLite](https://sqlite.org/) database. The database **must follow the expected schema**, please [follow this link](./cli/import/examples/example.sql) to see an example.
 
-The `import` command will use those file pairs to create a new [SQLite](https://sqlite.org/) or [PostgreSQL](https://www.postgresql.org/) database that will be used internally by the Annotation Tool. The destination database does not need to be empty, new imported file pairs can be added to previous imports.
```

Comment:
There is a discussion on [postgresql support](https://github.com/src-d/code-annotation/pull/206#discussion_r175986723) but at any rate, even if this PR breaks it - removing existing Posgresql support in this PR does not seem like a good idea to me.

If merging fast is required, I would propose create a new issue for implementing the support switching a type and then adding it asap, rather then having both - adding new feature and removing an old one in the same PR.

Reply:
Let me explain the current situation:
- I don't see the benefit of blocking the new feature (UASTs in FilePairs) till we fix a problem that we do not have yet (nobody is using PostgreSQL yet)
- If we don't block this PR, it would be merged, so the PostgreSQL support would be broken the facto. If we leave the docs (and the code) as they currently are, it could confuse people trying to use PostgreSQL from now on.
- I don't see the benefit of releasing something we already know it is broken.
- I'm not purposing to drop definitively the PostgreSQL support; It can be resumed considering the issue https://github.com/src-d/code-annotation/issues/222 . The commit dropping PostgreSQL support 94d41ab can be reverted; nothing is lost.